EmployeeAdmin/src/com/employee/Employee.java

```diff
@@ -6,29 +6,96 @@
  *
  */
 public abstract class Employee {
-
-	public String name;
-	public String email;
-	public int age;
+	
+	protected int id;
+	protected String name;
+	protected String email;
+	protected int age;
+	protected String employeeType;
 	
 	/**
 	 * Initialize a new employee
-	 * @param name The name of an employee
+	 * @param name The name of the employee
 	 * @param email The email of the employee
 	 * @param age The age of the employee
+	 * @param employeeType The employeeType of the employee
 	 */
 	
-	public Employee(String name, String email, int age) {
+	public Employee(int id, String name, String email, int age, String employeeType) {
+		//TODO assign the id dynamically
+		this.id = id;
 		this.name = name;
 		this.email = email;
 		this.age = age;
+		this.employeeType = employeeType;
 	}
 	
 	/**
 	 * Set the name to the employee
-	 * @param name The name of an employee
+	 * @param name The name of the employee
 	 */
 	public void setName(String name) {
 		this.name = name;
 	}
+	
+	/**
+	 * Set the email to the employee
+	 * @param email The email of the employee
+	 */
+	public void setEmail(String email) {
+		this.email = email;
+
+		//TODO write restrictions for the type of an email
+	}
+	
+	/**
+	 * Set the age to the employee
```

Comment:
consistency: set the age of the employee (as the next line - which is correct) - the same for the rest of the setter and getter methods